io-functions-pay-portal/GetPaymentInfo/index.ts

```diff
@@ -22,12 +23,13 @@ const app = express();
 secureExpressApp(app);
 
 // Add express route
-app.get("/api/v1/payment-requests/:rptId", HttpCtrl());
+app.get("/api/v1/payment-requests/:rptId", GetPaymentInfoCtrl(apiClient));
 
 const azureFunctionHandler = createAzureFunctionHandler(app);
 
 // Binds the express app to an Azure Function handler
 function httpStart(context: Context): void {
+  console.log("start..........httpStart....GetPaymentInfoCtrl");
```

Comment:
Remove this when PR is ready

Reply:
I apologize, you're right, it'll removed

Reply:
No need to apologize 👍